src/perl/Permabit/KernelModule.pm

```diff
@@ -51,7 +51,7 @@ sub load {
   my $machine = $self->{machine};
   my $modName = $self->{modName};
 
-  if ($self->SUPER::load()) {
+  if ($self->{useDnf} || $self->SUPER::load()) {
     # modprobe will tell us whether we managed to load successfully.
```

Comment:
What is this line even for?  SUPER::load() is there to tell us whether the requeseted kernel module has already been loaded or not, so we don't try to re-laod a module that is already loaded. This doesn't make sense to me.